code/ATMOSPHERICS/he_pipes.dm

```diff
@@ -145,4 +147,4 @@ obj/machinery/atmospherics/pipe/simple/heat_exchanging/junction
 			return
 
 		update_icon()
-		return
+		return
```

Comment:
Please always leave empty line at the end of file.

Reply:
Got it.